InitialCommit

```diff
@@ -0,0 +1,112 @@
+var sport = prompt("Time to get coaching advice on a sport!");
```

Comment:
Looks like this was saved as the file `InitialCommit`, would recommend saving it as a `.js` file extension so you get the added benefit of syntax highlighting!